src/Controls/src/Core/Compatibility/Handlers/ViewHandlerDelegator.cs

```diff
@@ -81,7 +82,9 @@ public void DisconnectHandler()
 #if WINDOWS
 			VisualElementRenderer<TElement, TNativeElement>.SetVirtualView(view, _viewHandler, onElementChanged, ref _element, ref _mapper, _defaultMapper, autoPackage);
 #else
-			VisualElementRenderer<TElement>.SetVirtualView(view, _viewHandler, onElementChanged, ref _element, ref _mapper, _defaultMapper, autoPackage);
+			VisualElementRenderer<TElement>.SetVirtualView(view, _viewHandler, onElementChanged, ref _tempElement, ref _mapper, _defaultMapper, autoPackage);
+			_element = _tempElement is null ? null : new(_tempElement);
+			_tempElement = null;
```

Comment:
While `SetVirtualView` is running, it appears that the getter for `Element` is called. Best I came up with is to make a `_tempElement` and clear it similar to:

https://github.com/dotnet/maui/blob/2c40a29d40911d28be620636bbae5f3152607ece/src/Controls/src/Core/Compatibility/Handlers/VisualElementRenderer.cs#L60

https://github.com/dotnet/maui/blob/2c40a29d40911d28be620636bbae5f3152607ece/src/Controls/src/Core/Compatibility/Handlers/VisualElementRenderer.cs#L298-L304